testclient/testclient.go

```diff
@@ -43,23 +45,75 @@ const (
 	Generate = 0
 	// Verify an attestatoin report (Mode)
 	Verify = 1
+	// Create a TLS connection with an exemplary connector = server (Mode)
+	TLSConn  = 2
 )
 
+/* Creates TLS connection between this client and a server and performs a remote
+ * attestation of the server before exchanging few a exemplary messages with it
+ */
+func TestTLSConn(connectoraddress, rootCACertFile string) {
+	var timeout = 10 * time.Second
+	// get server cert
+	servercert, err := ioutil.ReadFile(rootCACertFile)
+	if err != nil {
+		log.Error("[Testclient] Could find root CA cert file.")
+		return
+	}
+	roots := x509.NewCertPool()
+	success := roots.AppendCertsFromPEM(servercert)
+	if !success {
+		log.Error("[Testclient] Could not add servercert to root CAs.")
+		return
+	}
+	// Create TLS config with server as root CA
+	conf := &tls.Config{
```

Comment:
You could provide a type alias in the `attestedtls` package like `type Config = tls.Config` so that you could use the package as a drop-in-replacement with an imported name of `pls`.